packages/runtime/plugin-garfish/src/cli/index.ts

```diff
@@ -151,6 +151,11 @@ export const garfishPlugin = ({
                   .use(webpack.BannerPlugin, [{ banner: 'Micro front-end' }]);
               }
             },
+            webpack: webpackConfig => {
+              if (!webpackConfig.output?.uniqueName) {
```

Comment:
Can we use `tools.bundlerChain` to set this config? I think Rspack supports uniqueName too.

Reply:
https://www.rspack.dev/config/output.html#outputuniquename

Reply:
done，i forgot it, sorry